htdocs/core/boxes/box_services_contracts.php

```diff
@@ -96,12 +96,12 @@ function loadBox($max=5)
 					$contratlignestatic->id=$objp->cdid;
 					$contratlignestatic->fk_contrat=$objp->rowid;
 					$contratlignestatic->label=$objp->label;
-					$contratlignestatic->description=$objp->description;
+					$contratlignestatic->description=dol_trunc($objp->description);
```

Comment:
We should avoid using dol_trunc now into td fields and try to address truncation using css styles on td.
For example class="tdoverflowmax100 maxwidth100onsmartphone" should have effect of automatic trunction if longer than 100px